syntax = "proto3";
import "PacketsId.proto";

package my.proto.package;

message MousePositionMessage
{
	option (package_id) = MOUSE_POSITION_MESSAGE;
	float x = 1;
	float y = 2;
}

message MouseClickMessage
{
	option (package_id) = MOUSE_CLICK_MESSAGE;
}
